kubernetes/INSTALL.md

```diff
@@ -7,11 +7,11 @@
     
     kubectl config set-context `kubectl config view | grep current-context | awk '{print $2}'` --namespace <ns>
 
-**Install Istio infra**
+**Install Istio control plane and management plane**
 
```

Comment:
I was referring to an Software Defined Networking model, not to SNMP, this is 2017. Management plane does not access devices directly, this is an anti-pattern.

Maybe the api layer is not yet fully exposed in the Istio code, but the config is there, and the design contains the management plane, so I will not restrict the naming to control plane.

I will call it Istio core, since it contains the logically centralized components.